### DISCUSSION TITLE
profile file name and cycle number does not increase consistently

### DISCUSSION THREAD

**Discussion by vturpin** [OneArgo/ADMT#73] [General]
We’ve identified some floats where the cycle number in the file name does not increase consistently. These floats don’t follow the expected naming convention format modeWmoid_cycleNumber.nc, where the cycle number increments by 1 with each new cycle.

Here’s an example: WMO ID = "4902102".
![image](https://github.com/user-attachments/assets/0a601ecb-1239-406e-9b41-9601058e39ca)

Screenshot from [GDAC](https://data-argo.ifremer.fr/dac/aoml/4902102/profiles/).

I’m wondering if this is expected behavior or if it indicates an issue somewhere? So far, we've identified around 160 floats with this pattern, but further investigation may reduce that number since some of them only show a 2- or 3-cycle jump (which may be not so problematic).

Let me know your thoughts.

**Comment by tcarval** [comment]
Hello Victor, for good or bad reasons, a DAC may not receive data from one or more cycles.
In my opinion, it is the role of the delayed mode operator to check with the DAC that cycles are definitely missing or that they can be recovered with a reprocessing.

**Reply by vturpin** [reply]
Thanks Thierry, so I understand that a jump of serveral cycle can exist and we need to consider that situation in our computation.
However in some cases, like the exemple above, the number of cycle is extremely high. 3089 cycles is probably an error. 
Floats are not designed to sample more than 300 cycles (this would be 8 years in standard Argo config)
